grails-app/views/districtProposal/editModules/_editDistrict.gsp

```diff
@@ -47,6 +47,17 @@
                 </select>
             </div>
         </fieldset>
+        <fieldset class="form-group">
+            <label for="backerType" class="col-sm-2 col-md-1 control-label"><g:message code="kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.label"/>:</label>
+            <div class="col-sm-8 col-md-7">
+               <select class="form-control input-lg" name="backerType" >
+                   <option value="">---</option>
+                    <g:each in="${org.kuorum.rest.model.communication.participatoryBudget.BackerTypeRSDTO}" var="type">
+                        <option value="${type}" ${type==command.backerType?'selected':''}><g:message code="kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.${type}"/></option>
+                    </g:each>
+                </select>
```

Comment:
All of this code is already done. Check formUtil taglib. There is a utility that prints all this code using as arguments only the field and the  command.

Reply:
Example: 
<formUtil:selectEnum command="${command}" field="campaignVisibility" showLabel="false"/>